src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -166,14 +171,16 @@ public void changeInformationPanel(ChangeInformationPanelRequestEvent event) {
             return; // Short circuit if the current information panel is the same as the requested information panel
         } else {
             informationPanelPlaceholder.getChildren().removeAll(homePanel.getRoot(), personInformationPanel.getRoot(),
-                    helpPanel.getRoot());
+                    helpPanel.getRoot(), birthdayStatisticsPanel.getRoot());
 
             if (event.getPanelRequestEvent().equals(PERSON_INFORMATION_PANEL)) {
                 informationPanelPlaceholder.getChildren().add(personInformationPanel.getRoot());
             } else if (event.getPanelRequestEvent().equals((HOME_PANEL))) {
                 informationPanelPlaceholder.getChildren().add(homePanel.getRoot());
             } else if (event.getPanelRequestEvent().equals((HELP_PANEL))) {
                 informationPanelPlaceholder.getChildren().add(helpPanel.getRoot());
+            } else if (event.getPanelRequestEvent().equals((BIRTHDAY_STATISTICS_PANEL))) {
```

Comment:
Add testings for this under ChangeInformationPanelSystemTest, test for both birthday statistics command and alias.